apps/server/src/shared/domain/interface/permission.enum.ts

```diff
@@ -1,4 +1,5 @@
 export enum Permission {
+	TEST_ONLY = '___TEST___',
```

Comment:
Finde ich hilfreicher im Code. 
Ich hätte ein enum TestPermission extends Permission auch besser gefunden. 
Nur geht das leider nicht. Ich kann mir aber vorstellen das auch in den Client Type safe Test geschrieben werden wollen und es daher hier auch nicht ganz falsch aufgehoben ist.

Reply:
Sollte man sich die Permission nicht lieber im Test erzeugen, z.B. mit `const TEST_PERMISSION = 'TEST_PERMISSION' as Permission`? Es gehört doch in den Test, wenn es nur zum Testen ist.